x/leverage/keeper/keeper.go

```diff
@@ -146,12 +150,15 @@ func (k Keeper) Withdraw(ctx sdk.Context, supplierAddr sdk.AccAddress, coin sdk.
 
 		// Check for sufficient collateral
 		collateral := k.GetBorrowerCollateral(ctx, supplierAddr)
-		if collateral.AmountOf(coin.Denom).LT(amountFromCollateral) {
-			return sdkerrors.Wrap(types.ErrInsufficientBalance, coin.String())
+		if collateral.AmountOf(uToken.Denom).LT(amountFromCollateral) {
+			return types.ErrInsufficientBalance.Wrapf("%s balance + %s collateral / %s withdraw",
```

Comment:
Maybe make this err message a little more clear

Reply:
Done at [this commit](https://github.com/umee-network/umee/pull/1222/commits/9298521e7f3db01fa3bd7dbbc8671cac82b21ab2)